tests/hypothesis/test_fernet.py

```diff
@@ -8,6 +8,10 @@
 from cryptography.fernet import Fernet
 
 
+# Unlimited timeout will become the default in the future. When it does
+# we should remove this. See:
+# https://hypothesis.readthedocs.io/en/latest/settings.html?highlight=
+# timeout#hypothesis.settings.timeout
```

Comment:
Tiny nit: please don't include the highlight querystring

Reply:
tbf without it there's no way to pick out the timeout section (there's no anchor).